### PR TITLE
Fix perf sha

### PR DESCRIPTION
**Pull request by garyverhaegen-da**
This PR fixes the Slack notification on daily perf runs. It also updates the perf sha.

CHANGELOG_BEGIN
CHANGELOG_END

### CODE REVIEW

File: ci/daily_tell_slack.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 parameters:
-  success-message: '"$(Agent.JobName) passed: $COMMIT_LINK"'
+  success-message: '\"$(Agent.JobName) passed: $COMMIT_LINK\"'
```
Comment (cocreature) [inline ci/daily_tell_slack.yml:5]: I don’t understand the escaping here. iirc (and I did briefly look at the yaml spec to confirm) you only have to escape single quotes within a single-quoted string.

Comment (garyverhaegen-da) [reply ci/daily_tell_slack.yml:5]: They're escaped for Bash, not for YAML. I need the quotes in there to produce valid JSON.

**Comment by garyverhaegen-da** [comment]
It looks like the sha is not right. I took it from the Slack message, so either the code has changed again since then, or the Slack message does not provide the right sha.

**Comment by garyverhaegen-da** [comment]
Oh. Actually the issue is that this was merged after the daily run for today. I wonder why it took so long to get auto-merged. 🤷

**Comment by cocreature** [comment]
> I wonder why it took so long to get auto-merged. shrug

Afaik that’s just because I only approved this morning (sorry about that).

**Comment by garyverhaegen-da** [comment]
Ah. No problem; I was just puzzled there for a moment when I noticed we did not get the perf message today.

File: ci/daily_tell_slack.yml (hunk 1 above)
Comment (garyverhaegen-da) [reply ci/daily_tell_slack.yml:5]: Elaborating a little bit, we use this here:
```
MESSAGE="${{ parameters['success-message'] }}"
```
and want that to Azure-resolve to
```
MESSAGE="\"$(Agent.JobName) passed: $COMMIT_LINK\""
```
so we have quotes in there for when we call
```
PAYLOAD="{\"text\":$MESSAGE}"
```
so we produce valid JSON.

Now, you'd think we could have just kept the existing form of
```
PAYLOAD="{\"text\":\"$MESSAGE\n\"}"
```
to have explicit quotes at the point of use. The problem with that is that the alternative, for the perf job, is produced by `jq`, which has to produce a valid JSON string (so it includes all the relevant escaping), and that has embedded quotes. So we need the default case to have embedded quotes too.

Comment (cocreature) [reply ci/daily_tell_slack.yml:5]: Makes sense, thanks for the explanation!

**Pull request merged**